app/cdap/components/hydrator/components/TopPanel/NameAndDescription/styles.ts

```diff
@@ -104,7 +104,7 @@ export const HydratorMetadata = styled.div`
     height: 185px;
     cursor: auto;
     padding: 10px;
-    width: 100%;
+    width: 92%;
```

Comment:
why this change?

Reply:
it kinda solves the problem when you expand nameAndDescrption, it went out of the border